README.md

```diff
@@ -169,6 +177,43 @@ Found 2 known vulnerabilities in 1 packages
 ]
 ```
 
+## Security Model
+
+This section exists to describe the security assumptions you **can** and **must not**
+make when using `pip-audit`.
+
+TL;DR: **If you wouldn't `pip install` it, you should not `pip audit` it.**
+
+`pip-audit` is a tool for auditing Python environments for packages with
+*known vulnerabilities*. A "known vulnerability" is a flaw in a package that,
+if uncorrected, *might* allow a malicious actor to perform unintended actions.
+
+For example: `somepackage==1.2.3` might have a *known vulnerability* in the form
+of a deserialization bug. An attacker *might* be able to exploit that
+vulnerability, depending on how your code (or your other dependencies) use
+`somepackage`.
+
```

Comment:
```suggestion
```

I think we can nuke this.

Reply:
Yeah, I think it's already clear enough.